### PR TITLE
Add blog rss feed

### PR DESCRIPTION
**Pull request by aileenrose**
Includes both blog posts and podcasts.

Request: https://github.com/sourcegraph/about/issues/1662

### CODE REVIEW

**Comment by sqs** [comment]
awesome!

**Review by lorihey — APPROVED** [review]

**Comment by ryanslade** [comment]
Nice!

File: website/gatsby-config.js [hunk 1]
```diff
@@ -110,6 +110,67 @@ module.exports = {
...
+                  url: site.siteMetadata.siteUrl + '/' + edge.node.fields.blogType + '/' + edge.node.fields.slug,
+                  guid: site.siteMetadata.siteUrl + '/' + edge.node.fields.blogType + '/' + edge.node.fields.slug,
+                  custom_elements: [{ "content:encoded": edge.node.html }],
+                })
+              })
+            },
+            query: `
+              {
+                allMarkdownRemark(
+                  filter: { fields: { blogType: { in: ["blog","podcast"] } } }
```
Comment (vanesa) [inline website/gatsby-config.js:144]: We should make a note to update this once we introduce the blog categories.

**Review by vanesa — APPROVED** [review]

**Comment by pecigonzalo** [comment]
<3

**Pull request merged**